### PR TITLE
Project Timezone API #9871

### PR DESCRIPTION
**Pull request by anatol-sialitski**

### CODE REVIEW

File: modules/core/core-api/src/main/java/com/enonic/xp/project/CreateProjectParams.java [hunk 1]
```diff
@@ -75,6 +80,11 @@ public boolean isForceInitialization()
         return forceInitialization;
     }
 
+    public String getTimezone()
```
Comment (rymsha) [inline modules/core/core-api/src/main/java/com/enonic/xp/project/CreateProjectParams.java:83]:
```suggestion
    public TimeZone getTimeZone()
```

**Review by rymsha — CHANGES_REQUESTED** [review]

File: modules/core/core-api/src/main/java/com/enonic/xp/project/CreateProjectParams.java (hunk 1 above)
Comment (anatol-sialitski) [reply modules/core/core-api/src/main/java/com/enonic/xp/project/CreateProjectParams.java]: After discuss we decided to use `ZoneId`

File: modules/core/core-project/src/main/java/com/enonic/xp/core/impl/project/ProjectServiceImpl.java [hunk 2]
```diff
@@ -466,6 +467,7 @@ private PropertyTree createProjectData( final CreateProjectParams params )
 
         projectData.setString( ProjectConstants.PROJECT_DESCRIPTION_PROPERTY, params.getDescription() );
         projectData.setString( ProjectConstants.PROJECT_DISPLAY_NAME_PROPERTY, params.getDisplayName() );
+        projectData.setString( ProjectConstants.PROJECT_TIMEZONE_PROPERTY, Objects.toString( params.getTimeZone(), null ) );
```
Comment (rymsha) [inline modules/core/core-project/src/main/java/com/enonic/xp/core/impl/project/ProjectServiceImpl.java:470]: don't set if null

File: modules/lib/lib-project/src/main/java/com/enonic/xp/lib/project/CreateProjectHandler.java [hunk 3]
```diff
@@ -135,4 +139,9 @@ public void setSiteConfig( final ScriptValue value )
         final List<SiteConfig> configs = buildSiteConfigs( value );
         this.siteConfigs = configs != null ? SiteConfigs.from( configs ) : null;
     }
+
+    public void setTimeZone( final String timeZone )
+    {
+        this.timeZone = timeZone != null ? ZoneId.of( timeZone, ZoneId.SHORT_IDS ) : null;
```
Comment (rymsha) [inline modules/lib/lib-project/src/main/java/com/enonic/xp/lib/project/CreateProjectHandler.java:145]: don't use SHORT_IDS

**Review by rymsha — CHANGES_REQUESTED** [review]

**Review by rymsha — APPROVED** [review]

**Pull request merged**